litmus/compile.ml

```diff
@@ -57,6 +57,7 @@ module Generic (A : Arch_litmus.Base)
         | MiscParser.TyDefPointer  -> pointer
         | MiscParser.Ty t -> Base t
         | MiscParser.Atomic t -> Atomic (Base t)
+        | MiscParser.Address -> base
```

Comment:
This can be problematic in the future, as base is `int`, while addresses can take concrete values up to 64bits. However, as  addresses numerical values should probably be ignored by litmus, it probably does not harm. Perhaps using `intptr_t` ?

Reply:
changed to `pointer`